### ISSUE TITLE
Cross-user syncing issues

### ISSUE THREAD

**Issue by jonahgreenthal** [mostafab/neg5#68]
When user A makes certain types of changes while user B is also logged in, under certain circumstances (details unclear) the changes may not show up for user B for a while. One example of this seems to be adding a phase.

**Comment by mostafab** [comment]
As user B, are you trying to see these changes without reloading the page? That capability isn't currently supported.

**Comment by jonahgreenthal** [comment]
Yes. Because of the SPA behavior, it's not at all obvious that refreshing would be required, so user B will generally not think to do so until already having been confused by this behavior.
